src/data/data-visualisation/accessibility/a11y.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: article
+title: "Data visualisation accessibility"
+description: "Why do we need to make our products accessible?"
+status: REVIEW
+tags: accessible-data
+order: 1
+---
+## Why do we need to make our products accessible?  
+
+Legally as part of the public sector we are required by law to meet accessibility requirements for both internal and external users.  
```

Comment:
'Legally' and 'required by law' say the same thing. I'd remove the first 'legally'.